etc/SuperBuild/cmake/Modules/BuildFFmpeg.cmake

```diff
@@ -12,12 +12,23 @@ else()
     set(FFmpeg_CXXFLAGS)
     set(FFmpeg_OBJCFLAGS)
     set(FFmpeg_LDFLAGS)
+
     if(APPLE AND CMAKE_OSX_DEPLOYMENT_TARGET)
         list(APPEND FFmpeg_CFLAGS "--extra-cflags=-mmacosx-version-min=${CMAKE_OSX_DEPLOYMENT_TARGET}")
         list(APPEND FFmpeg_CXXFLAGS "--extra-cxxflags=-mmacosx-version-min=${CMAKE_OSX_DEPLOYMENT_TARGET}")
         list(APPEND FFmpeg_OBJCFLAGS "--extra-objcflags=-mmacosx-version-min=${CMAKE_OSX_DEPLOYMENT_TARGET}")
         list(APPEND FFmpeg_LDFLAGS "--extra-ldflags=-mmacosx-version-min=${CMAKE_OSX_DEPLOYMENT_TARGET}")
     endif()
+    if(TLRENDER_VPX)
+        list(APPEND FFmpeg_CONFIGURE_ARGS
+            --enable-libvpx)
+        #
+        # Makre sure we pick the static libvpx we compiled, not the system one
```

Comment:
Typo: Makre -> Make

Using CMAKE_PREFIX_PATH won't work here, it can contain multiple paths.

Reply:
Should I rely on CMAKE_INSTALL_PREFIX instead?

Reply:
I think that would work but try it out to make sure.